### DISCUSSION TITLE
How to troubleshoot high cpu usage

### DISCUSSION THREAD

**Discussion by saeedesmaili** [coollabsio/coolify#2928] [Q&A] [answered]
I'm using a basic vps from Hetzner to host my Coolify instance, and I currently have a Plausible service running on it. The server  has a consistent cpu usage higher than 50%, sometimes even higher than 80% for hours.

<img width="203" alt="image" src="https://github.com/user-attachments/assets/8fa876f8-5478-4969-8b86-964b6374e119">

The Plausible's script is only added to my blog, which gets 20-30 visits per day.

Is this usual with Coolify, or is there something wrong with my instance? How can I troubleshoot this and find out what's causing this amount of CPU usage?

<img width="1372" alt="image" src="https://github.com/user-attachments/assets/81682e85-46fa-428e-b5f2-401981875ab2">

**Comment by peaklabs-dev** [comment, marked as answer]
@saeedesmaili There was a bug that could cause high CPU usage. Please upgrade to the latest version and try again.

**Comment by pencilcheck** [comment]
I would recommend using docker monitoring tools like dozzle to get a sense of the containers that are going rogue and it also aggregate logs from docker as well, not 100% useful but at least it helps a bit.

**Comment by LucaNerlich** [comment]
Ran into the same issue. Plausible uses "clickhouse" database which for some reason has fairly high cpu usage, when ran in docker. I was not able to tame it, even with the suggested workaround of setting the log level etc. 

If you need a low footprint analytics solution, just use "umami". The coolify one click template works beautifully.
